src/Forms/Shared/Samples/Network analysis/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml

```diff
@@ -0,0 +1,79 @@
+    <Grid>
+        <TableView
+            x:Name="ConfigureTable"
+            HasUnevenRows="True"
+            Intent="Settings">
+            <TableRoot>
+                <TableSection Title="Trace options">
+                    <SwitchCell
+                        On="True"
+                        OnChanged="IncludeBarriersChanged"
+                        Text="Include barriers" />
+                    <SwitchCell
+                        On="True"
+                        OnChanged="IncludContainersChanged"
+                        Text="Include containers" />
+                    <!--  NOTE that There are also: IncludeContent, IncludeStructures, IgnoreBarriersAtStartingPoints, ValidateConsistency  -->
+                </TableSection>
+                <TableSection>
+                    <ViewCell>
+                        <Label FontSize="Medium" Text="Example barrier condition for this data: 'Transformer Load' Equal '15'" />
+                    </ViewCell>
+                </TableSection>
+                <TableSection Title="Define new condition">
+                    <ViewCell>
+                        <Grid>
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="auto" />
+                                <RowDefinition Height="auto" />
+                                <RowDefinition Height="auto" />
+                                <RowDefinition Height="auto" />
+                            </Grid.RowDefinitions>
+                            <Picker
+                                x:Name="ComparisonSources"
+                                Grid.Row="0"
+                                ItemDisplayBinding="{Binding Name}"
+                                SelectedIndexChanged="OnComparisonSourceChanged" />
+                            <Picker x:Name="ComparisonOperators" Grid.Row="1" />
+                            <Picker
+                                x:Name="ComparisonValueChoices"
+                                Grid.Row="2"
+                                ItemDisplayBinding="{Binding Name}" />
+                            <Entry
+                                x:Name="ComparisonValue"
+                                Grid.Row="2"
+                                IsVisible="False" />
+                            <Button
+                                Grid.Row="3"
+                                Clicked="OnAddCondition"
+                                Text="Add condition" />
+                        </Grid>
+                    </ViewCell>
+                </TableSection>
+                <TableSection Title="Barrier conditions">
+                    <ViewCell>
+                        <ScrollView HeightRequest="150">
+                            <Label
+                                x:Name="ConditionBarrierExpression"
+                                Grid.Row="0"
+                                FontSize="Medium"
```

Comment:
I think setting a Bold font might be better than font size here. The large fonts feel a bit jarring.